### ISSUE TITLE
[question] load translation client side for dynamic routes

### ISSUE THREAD

**Issue by EyadAlghamdi** [vinissimus/next-translate#815]
hi, i have a dynamic route `/items/[id].ts` which i understand will break the static site caching and will use `getServerSideProps` by default, which i am trying to avoid using, since i want my pages to load fast,
so the other option is to use a `getStaticPaths`  like so
```ts
export const getStaticPaths: GetStaticPaths = () => {
  return {
    paths: [],
    fallback: "blocking"
  };
}
```
but the problem with this approach is that nextjs will create a huge number of cache files per each new item, which isn't ideal for me since we got thousands of items. (and each new item will use SSR anyway)

so the final solution I can come up with is to load the translations client-side instead of serverside (only for dynamic routes at least). is there a way for me to load these namespaces in the browser clientside?

**Comment by aralroca** [comment]
Yes you can use [DynamicNamespaces](https://github.com/vinissimus/next-translate#dynamicnamespaces)
